### PR TITLE
Rwa 2219

### PR DESCRIPTION
**Pull request by burale**
### JIRA link (if applicable) ###
[RWA-2219](https://tools.hmcts.net/jira/browse/RWA-2219)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/CleanUpSensitiveLogsServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        List<TaskFilter<?>> taskFilters = createTaskFilters(KEY, timestamp);
+        TaskOperationRequest request = new TaskOperationRequest(
+            TaskOperation.builder()
+                .type(TaskOperationType.CLEANUP_SENSITIVE_LOG_ENTRIES)
+                .runId("")
+                .build(),
+            taskFilters
+        );
+
+        doThrow(new IllegalArgumentException("cleanup exception"))
```
Comment (BenNzewi) [inline src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/CleanUpSensitiveLogsServiceTest.java:103]: Is this the right exception to throw when unable to connect to db?

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerReconfigureTaskConsumerTest.java [hunk 2]
```diff
@@ -29,8 +29,8 @@ public class TaskManagerReconfigureTaskConsumerTest extends SpringBootContractBa
     private static final String WA_RECONFIGURE_TASK_BY_ID = WA_URL + "/operation";
 
     @Test
-    @PactTestFor(pactMethod = "markTaskToReconfigureById204", pactVersion = PactSpecVersion.V3)
-    void testMarkTaskToReconfigureById204Test(MockServer mockServer) {
+    @PactTestFor(pactMethod = "markTaskToReconfigureById200", pactVersion = PactSpecVersion.V3)
+    void testMarkTaskToReconfigureById200Test(MockServer mockServer) {
```
Comment (BenNzewi) [inline src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerReconfigureTaskConsumerTest.java:33]: Pact test is broken. We have updated response status to 200 from 204 but actual endpoint n controller is still 204.

File: src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/CleanUpSensitiveLogsServiceTest.java (hunk 1 above)
Comment (raghera) [reply src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/CleanUpSensitiveLogsServiceTest.java:103]: Looks like it has been changed as part of this PR to OK to comply with the API docs: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1552143414#WATaskManagement:POST/task/operation(internal)-SuccessfulResponses

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerReconfigureTaskConsumerTest.java (hunk 2 above)
Comment (raghera) [reply src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerReconfigureTaskConsumerTest.java:33]: Looks like it has been changed as part of this PR to OK to comply with the API docs: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1552143414#WATaskManagement:POST/task/operation(internal)-SuccessfulResponses

File: src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/CleanUpSensitiveLogsServiceTest.java (hunk 1 above)
Comment (raghera) [reply src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/CleanUpSensitiveLogsServiceTest.java:103]: Created this ticket to cover this comment: https://tools.hmcts.net/jira/browse/RWA-2638

**Review by Lee365 — DISMISSED** [review]

**Pull request merged**